lib/onigumo/parser.ex

```diff
@@ -0,0 +1,14 @@
+defmodule Onigumo.Parser do
+  @moduledoc """
+  Web scraper
+  """
+
+  def main(root_path) do
+    root_path
+    |> File.ls!()
+    |> Enum.reject(&File.dir?(&1))
+    |> Enum.reject(&String.contains?(&1, "."))
```

Comment:
It means for hidden files or suffix?

Reply:
Exactly. This removes all hidden files from the file listing. In addition, it removes `.` and `..` entries that are not real files.